server/app/routes/poll.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+var router = require('express').Router()
+var db = require('../../database')
+var Poll = db.model('poll')
+var PollAnswer = db.model('pollAnswer')
+var Lecture = db.model('lecture')
+module.exports = router
+
+router.param('pollId', (req, res, next, id) => {
+  Poll.findOne({where:{id:id}, include: [{model:PollAnswer}]})
+  .then((poll) => {
+    if (!poll) res.sendStatus(404)
+    req.poll = poll
+    next()
+  })
+  .catch(next)
+})
+
+router.get('/', (req, res, next) => {
+  Poll.findAll({where:{lectureId:req.body.lectureId}})
```

Comment:
not sure if req.body can get sent in GET request... maybe can send as a parameter instead